code/FulpstationCode/deputy.dm

```diff
@@ -38,7 +38,14 @@
 	sensor_mode = SENSOR_COORDS
 	random_sensor = FALSE
 	fulp_item = TRUE
-	mutantrace_variation = DIGITIGRADE_STYLE
+	mutantrace_variation = MUTANTRACE_VARIATION
+
+/obj/item/clothing/under/rank/security/mallcop/skirt
```

Comment:
This file should really be placed in the same folder structure as it is in the parent folder. This ensures everything stays easy to find and consistent.

Reply:
Simple enough, sure thin.